Modules/_multiprocessing/semaphore.c

```diff
@@ -9,7 +9,7 @@
 
 #include "multiprocessing.h"
 
-#define USE_UNNAMED_SEMAPHORES
+#define USE_UNNAMED_SEMAPHORES // can't share named semaphore across fork, so use unnamed instead
```

Comment:
Add `#ifdef __PASE__` around it and move to around line 182. Also can define this in the `#ifdef MS_WINDOWS` block below. This simplifies the handling around sem_open in _rebuild.